addons/mail/static/src/core/common/composer.xml

```diff
@@ -111,8 +119,20 @@
                 </div>
                 <div t-else="" class="d-flex align-items-center gap-1 w-100 pe-2">
                     <div class="pt-1" t-att-class="{ 'mt-2': !props.composer.message }">
-                        <span t-if="props.composer.message" class="text-muted px-1 small" t-out="CANCEL_OR_SAVE_EDIT_TEXT" t-on-click="onClickCancelOrSaveEditText"/>
-                        <t t-else="">
+                        <span t-if="props.composer.message and !ui.isSmall" class="text-muted px-1 small" t-on-click="onClickCancelOrSaveEditText">
+                            <samp>Escape</samp>
+                            <em>
+                                to <a role="button" href="#" t-att-data-type="EDIT_CLICK_TYPE.CANCEL">cancel</a>
+                            </em>,
+                            <samp>
+                                <t t-if="env.inChatter">CTRL-Enter</t>
+                                <t t-else="">Enter</t>
+                            </samp>
+                            <em>
+                                to <a role="button" href="#" t-att-data-type="EDIT_CLICK_TYPE.SAVE">save</a>
+                            </em>
+                        </span>
```

Comment:
This change is wrong, we have done it in the JS for a reason.

We want this to be a phrase, this is completely untranslatable.

Reply:
Yes, I said in comment it's fine to do in template only for the `isSmall` part, not the whole getter. This is a misunderstanding from https://github.com/odoo/odoo/pull/213943#discussion_r2348934904 the `CANCEL_OR_SAVE_EDIT_TEXT` from `!ui.isSmall` but be kept in JS with `t-out`.